keanu-project/src/main/java/io/improbable/keanu/algorithms/variational/Optimizer.java

```diff
@@ -26,7 +49,7 @@
         return point;
     }
 
-    protected long totalNumLatentDimensions(List<? extends Vertex<DoubleTensor>> continuousLatentVertices) {
+    static long totalNumberOfLatentDimensions(List<? extends Vertex> continuousLatentVertices) {
```

Comment:
`Vertex<?>` is preferable to `Vertex`

Reply:
Basically because `Vertex` is interpreted as `Vertex<Object>` which will lead to compiler errors elsewhere.

Reply:
also in https://github.com/improbable-research/keanu/pull/131/files